<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogoDinamico</class>
 <widget class="QDialog" name="DialogoDinamico">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>330</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculadora de medias, máximos y mínimos</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="grupoNotas">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>321</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Notas</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMedia">
   <property name="geometry">
    <rect>
     <x>318</x>
     <y>264</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Calcular Media</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMedia">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Media</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMaximo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Máximo</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMinimo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Mínimo</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMedia">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMaximo">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMinimo">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogoDinamico</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogoDinamico</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
